src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -109,6 +113,22 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     disposeOnUnmount(this, [
       reaction(() => toJS(this.userSettings), settings => storage.set(settings)),
     ]);
+
+    if (props.isResizable && props.tableId) {
+      const columnConfig = Object.entries(userStore.preferences.storeTableConfig).find((element)=> {return element[0] == props.tableId;})[1];
+
+      if (columnConfig) {
+        Object.entries(columnConfig).map((value) => {
+          const outerValue = value;
+          const columnIndex = props.renderTableHeader.findIndex((value) => {
+            return value.className == outerValue[0];});
+
+          if (columnIndex >= 0 && outerValue[1]) {
+            this.columnswidth[columnIndex] = Number(outerValue[1]);
+          }
+        });
+      }
+    }
```

Comment:
If we switch to using an map (or object) then there is no need to do all this checking, we can just do `this.columnWidth[col.className]` or something similar.

```suggestion
      this.columnsWidth = columnConfig;
```